repo/packages/H/helloworld_invalid/0/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "packagingVersion": "4.0",
+  "name": "helloworld_invalid",
```

Comment:
Let's be consistent. `helloworld-invalid` instead of `helloworld_invalid`. You need to change the directory name too.